astroquery/mast/cutouts.py

```diff
@@ -117,36 +120,64 @@ def get_sectors(self, coordinates=None, radius=0*u.deg, objectname=None):
         coordinates : str or `astropy.coordinates` object, optional
             The target around which to search. It may be specified as a
             string or as the appropriate `astropy.coordinates` object.
-            One and only one of coordinates and objectname must be supplied.
+            One and only one of coordinates, objectname, and moving_target must be supplied.
         radius : str, float, or `~astropy.units.Quantity` object, optional
             Default 0 degrees.
             If supplied as a float degrees is the assumed unit.
             The string must be parsable by `~astropy.coordinates.Angle`. The
             appropriate `~astropy.units.Quantity` object from
             `astropy.units` may also be used.
+            This argument is ignored if moving_target is supplied.
         objectname : str, optional
             The target around which to search, by name (objectname="M104")
             or TIC ID (objectname="TIC 141914082").
-            One and only one of coordinates and objectname must be supplied.
+            One and only one of coordinates, objectname, moving_target must be supplied.
+        moving_target : str, optional
```

Comment:
Why do we need a new kwarg for moving targets? At the end of the day they are the name of the object to query, and from the user's pow I don't see a difference whether querying for M1 or Ceres.

Reply:
Hi @bsipocz , I've been tasked with addressing threads for this PR so it can get merged: From what I can tell it looks like there are two separate databases, and setting `moving_target` will make the request call for the moving target database in MAST.

Reply:
Hi, so my reason for doing this was twofold:
- The resolving method for moving vs regular targets is completely different, and naming in astronomy is not always unique so I was worried about the user putting in a valid moving target and getting a stationary result.
- The output for moving targets is not quite the same as for regular targets, since it's a different type of cutout, so I thought it was clearer to keep the two queries more separate.

Both of these concerns could be addressed by having a moving target flag that defaults to false instead of a whole new argument.

Reply:
The naming mess affects not just moving targets, I'm sure there is overlap within non-moving ones if we look closely, too, and also, the users don't need to know what's under the hood, that depending on the type of the object the query goes to a different database. 
So, I find the logic of having a bool `moving_target` clearer than basically having two separate APIs for the name depending on the service.

Reply:
I've made a new commit that turns `moving_target` into an optional bool argument. With your approvals, I'll make another commit to update the docstrings, tests, documentation, and collapse the commits into one.